README.md

```diff
@@ -135,22 +376,119 @@ batch(() => {
 console.log('Process end');
 ```
 
-With the following output:
+output:
 
 ```text
 Arsène Lupin
 Sherlock Holmes
 Process end
 ```
+> [!NOTE]
+> - Managing stores in a batch context can be quite challenging (getting store values inside a batch, subscribing in a batch, avoiding rerunning calculations outside the batch if the value has been updated inside, ...). All potential situations have been specified in the specs.
+> - `batch` can be called inside `batch`. The subscriber calls are performed at the end of the first batch, synchronously.
+> - Retrieving the immediate value of a store (using any of the three methods mentioned earlier: calling `subscribe` with a subscriber that will be called with the value synchronously, using `get` or calling the store as a function) always provides the value based on the up-to-date values of all dependent stores (even if this requires re-computations of a computed or a derived inside the batch)
+> - all calls to subscribers (excluding the first synchronous call during the subscribe process) are deferred until the end of the batch
+> - if a subscriber has already been notified of a new value inside the batch (for example, when it is a new subscriber registered within the batch), it won't be notified again at the end of the batch if the value remains unchanged. Subscribers are invoked only when the store's value has changed since their last call.
 
-## Installation
 
-You can add Tansu to your project by installing the `@amadeus-it-group/tansu` package using your favorite package manager, ex.:
+### asReadable
 
-* `yarn add @amadeus-it-group/tansu`
-* `npm install @amadeus-it-group/tansu`
+[api documentation](https://amadeusitgroup.github.io/tansu/functions/asReadable.html)
 
-## Usage
+`asReadable` transforms a store into a readable one. A readable store is a store without the set and update method.
```

Comment:
Actually, this is not rigorously exact. Technically, a writable store (i.e. implementing the `WritableSignal` interface) is also  a readable store (i.e. it implements the `ReadableSignal` interface).